atlasdb-container-test-utils/src/main/resources/docker-compose-cassandra-three-node.yml

```diff
@@ -5,6 +5,8 @@ services:
     image: palantirtechnologies/docker-cassandra-atlasdb:$CASSANDRA_VERSION
     environment:
       - CASSANDRA_SEEDS=cassandra1,cassandra2,cassandra3
+      - MAX_HEAP_SIZE=$CASSANDRA_MAX_HEAP_SIZE
+      - HEAP_NEWSIZE=$CASSANDRA_HEAP_NEWSIZE
```

Comment:
Just out of curiosity, why do we override these values. If they are recommended values, should we also modify the docker images to be built with this configuration. Also, do you have a (intutive)measure of whether this save us time?